airbyte-webapp/src/packages/cloud/lib/auth/GoogleAuthService.ts

```diff
@@ -89,6 +92,15 @@ export class GoogleAuthService implements AuthService {
     return updateEmail(user, email);
   }
 
+  async changePassword(passwd: string, newPassword: string): Promise<void> {
+    const user = await this.getCurrentUser()!;
+
+    const credential = EmailAuthProvider.credential(user.email!, passwd);
```

Comment:
We try to avoid `!` even though we have it in some places.

Is it possible for this email to be empty? if yes - lets check it for null and through error otherwise. Credential should always be a email, right?